src/lightning-talks.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: content.liquid
+title: Lightning Talks
+---
+
+## What's a Lightning Talk?
+
+Lightning talks are short presentations, limited to 5 minutes, on any topic that might interest the Python community. It doesn't have to be something you've created, feel free to share something you've learned or a technique you think others will find useful.
+
+- Is there something at work that everyone asks for your help with? Talk about that!
+- Did you recently learn something that made a difference for you? Talk about that!
+- Is there a topic you've always wanted to understand better? Talk about that!
+
+Slides are optional but encouraged!
+
+## Things people say when they think they can't do a lightning talk
+
+- **"Everyone already knows THING_X"**
+    - No, they don't.
+    - They think they do, but you have a better understanding than they do, and they will learn something.
+    - Even if they do, it's interesting to hear another person's explanation.
+- **"I’m not an expert at anything"**
+    - You know more than you think you do.
+    - You don't have to be an expert, you just have to be interested enough to talk for 5 minutes.
+    - You can do a 1-minute lightning talk if you want.
+- **"It's scary talking in front of people"**
+    - Yes, but you will feel really accomplished when you've done it.
+    - Folks at PyCon are super-friendly and is always welcoming of new speakers.
+    - Everyone is nervous speaking in public, even people you think are not.
+    - [You can give a lightning talk without talking!](https://www.youtube.com/watch?v=Aio0whC1iYg#t=25m58s)
+- **"I have an idea but it's no good"**
+    - It's a better idea than you think.
+    - Are there really any bad ideas for lightning talks? :)
+
+## Lightning Talks at PyCon India 2024
+
+Please refer to the schedule for Lightning Talk slots. Registration opens on the day of the conference, and due to the likely high number of registrations, a selection process will determine which talks are accepted. You will be notified if your talk is selected.
```

Comment:
We should write more about the registration and selection process.

Reply:
Mentioned about the google form and it will be on a first-come, first-served basis. 
Is it fine? @naba